tests/docs/running-e2e-test.md

```diff
@@ -38,12 +41,16 @@ E2E tests are designed for verifying the functional correctness by replicating e
 
 If you are starting from scratch and just want to build dapr, deploy it, and run the e2e tests to your kubernetes cluster, do the following:
 
-1. Remove the test namespace, if it exists.
+1. Remove the test namespace, if it exists
 *Make sure you have DAPR_NAMESPACE set properly before you do this!*
 ```bash
 make delete-test-namespace
 ```
-2. Build, deploy, run tests from start to finish
+2. Delete the mutatingwebhook configurations, if it exists
```

Comment:
Does it would hung in local dev?

Reply:
At local still can clean resource, why not do that?

Reply:
Use helm uninstall command to clean resource?  The steps in the document did not delete the mutatingwebhook config. This cause "tls: bad certificate" problem when injector daprd sidecar

Reply:
This is a good point, we should probably just have helm uninstall here. The doc is just sort of assuming you've cleaned up appropriately. Suggest adding helm uninstall here instead of kubectl delete ...